tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13019,6 +13019,26 @@ build_unused_global_define_from_params(void)
   return;
 }
 
+/**
+   \brief Helper function: test if the param length exists as compiler created
+          symbol which represents length of any assumed length string argument
+          in the arg list
+   \param length is the compiler created symbol which represents length of
+          assumed length string argument
+ */
+INLINE static bool
+cg_fetch_clen_parent(SPTR length)
```

Comment:
Nit 1: A better position for this function may be next to `cg_fetch_clen_parampos`.

Nit 2: If the function is named `cg_fetch_clen_parent` then it makes more sense for it to return the `SPTR` for the parent if the parent can be found in the argument list, or `SPTR_NULL` if it cannot be found.

Reply:
@bryanpkc ,Thanks a lot for your review comments, i have addressed both of them

Reply:
@bryanpkc Hi, i have addressed minor or Nit comments, could you please look at them once, thanks